src/main.rs

```diff
@@ -324,7 +324,16 @@ fn start_vmm(cmd_arguments: ArgMatches) {
     } else {
         SeccompAction::Trap
     };
-    let hypervisor = hypervisor::new().unwrap();
+    let hypervisor = match hypervisor::new() {
+        Ok(h) => h,
+        Err(e) => {
+            eprintln!(
+                "Failed to open hypervisor interface (is /dev/kvm available?): {:?}",
```

Comment:
Should we mention Hyper-V? Or make this message a bit more generic?

Reply:
Don't worry too much about this. We will cross the bridge when we get there.

Reply:
Sounds good!

Reply:
> Should we mention Hyper-V? Or make this message a bit more generic?

Yeh, I was cognisant of this when I wrote it (that's why the first half is more generic). Ideally the error message coming from the `Kvm::new()` would be better